third_party/silabs/SiWx917_sdk.gni

```diff
@@ -256,7 +256,7 @@ template("siwx917_sdk") {
     }
 
     defines += [ "LWIP_NETIF_API=1" ]
-    if (lwip_ipv4) {
+    if (chip_enable_wifi_ipv4) {
```

Comment:
Validated for all the combinations, such as enabling and disabling both IPv4 and IPv6?

Reply:
validated all the combinations